src/julia-parser.scm

```diff
@@ -8,7 +8,7 @@
 ;; be an operator.
 (define prec-assignment
   (append! (add-dots '(= += -= *= /= //= |\\=| ^= ÷= %= <<= >>= >>>= |\|=| &=))
-           '(:= => ~ $=)))
+           '(:= => ~ $= ⊻=)))
```

Comment:
`⊻=` can be under add-dots now right? I think `$=` wasn't added there mostly because `$` has other meanings that were making it complicated to parse correctly